client/trino-client/src/main/java/io/trino/client/ClientSession.java

```diff
@@ -35,6 +35,7 @@
 {
     private final URI server;
     private final String user;
+    private final String requestedUser;
```

Comment:
let's `sessionUser`, but perhaps wait a bit if others have different opinin

Reply:
For ClientSession -- it agree.

For JDBC driver configuration, the `user` and `password` properties are the facto standard way of configuring authentication (principal, and identity on the server).
I believe at TrinoDriverUri and ConnectionProperties level the user should mean authentication principal and eg sessionUser needs to mean the impersonated person.
Which may or may not imply the naming back into ClientSession